packages/agents/.gitignore

```diff
@@ -1,2 +1 @@
-__pycache__
```

Comment:
Also, raises some questions about why we might need `__pycache__` dirs.

Reply:
AFAIK those are generated automatically, hence they were added to .gitignore. WDYM?

Reply:
Ohh sorry, I'm kinda exhausted today. On the top of my mind, I had a question "Why do we remove that line from `.gitignore`? That's what puzzles me 🤗.

Reply:
Oh, that's what you meant! I just moved it to a root `.gitignore`, so those dirs are still (but definitely) ignored :)

Reply:
No worries, and once again thanks for the explanation.